pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.ejs

```diff
@@ -26,7 +26,12 @@
                 <th></th>
                 <th>QID</th>
                 <th>Points</th>
-                <th>Number of Submissions</th>
+                <th>Total Submissions</th>
```

Comment:
Do we care about this?

Reply:
I don't think so.

This is how the view will look now:
![image](https://user-images.githubusercontent.com/7872295/109556200-5b8de200-7a8b-11eb-8d0e-362566c0c32e.png)

I pulled out anything unnecessary.